File: src/LeetCode/Algorithms/Defanging_an_IP_Address.java
package LeetCode.Algorithms;

// https://leetcode.com/problems/defanging-an-ip-address/
public class Defanging_an_IP_Address {
	public static void main(String[] args){
		String address = "255.100.50.0"; // "1.1.1.1";

		System.out.println(defangIPaddr(address));
	}

	public static String defangIPaddr(String address) {
		return address.replace(".", "[.]");
	}
}

File: src/LeetCode/Algorithms/JavaScript/Defanging an IP Address.js
/**
 * https://leetcode.com/problems/defanging-an-ip-address/
 * @param {string} address
 * @return {string}
 */
var defangIPaddr = function(address) {
	return address.replace(/\./gi, '[.]');
};

let address = '255.100.50.0';

console.log(defangIPaddr(address));